### DISCUSSION TITLE
Config without docker

### DISCUSSION THREAD

**Discussion by guiwestrup** [adalessa/laravel.nvim#52] [Q&A]
I'm installing with /AstroNvim/AstroNvim, I put the configuration in the core.lua file:
`{
    "adalessa/laravel.nvim",
    dependencies = {
      "nvim-telescope/telescope.nvim",
      "tpope/vim-dotenv",
      "MunifTanjim/nui.nvim",
    },
    cmd = { "Sail", "Artisan", "Composer", "Npm", "Yarn", "Laravel" },
    keys = {
      { "<leader>la", ":Laravel artisan<cr>" },
      { "<leader>lr", ":Laravel routes<cr>" },
      { "<leader>lm", ":Laravel related<cr>" },
      {
        "<leader>lt",
        function()
          require("laravel.tinker").send_to_tinker()
        end,
        mode = "v",
        desc = "Laravel Application Routes",
      },
    },
    event = { "VeryLazy" },
    config = function()
      require("laravel").setup()
      require("telescope").load_extension "laravel"
    end,
  }`

But when opening a laravel project, it is generating the following error:

`Failed to run `config` for laravel.nvim

.../.local/share/nvim/lazy/plenary.nvim/lua/plenary/job.lua:107: docker: Executable not found
stack traceback:
	.../.local/share/nvim/lazy/plenary.nvim/lua/plenary/job.lua:107: in function 'new'
	...are/nvim/lazy/laravel.nvim/lua/laravel/runners/async.lua:28: in function <...are/nvim/lazy/laravel.nvim/lua/laravel/runners/async.lua:8>
	.../nvim/lazy/laravel.nvim/lua/laravel/application/init.lua:85: in function 'run'
	...al/share/nvim/lazy/laravel.nvim/lua/laravel/commands.lua:48: in function 'load'
	.../nvim/lazy/laravel.nvim/lua/laravel/application/init.lua:65: in function 'warmup'
	....local/share/nvim/lazy/laravel.nvim/lua/laravel/init.lua:26: in function 'setup'
	/Users/guilhermewestrup/.config/nvim/lua/plugins/core.lua:162: in function 'config'
	...local/share/nvim/lazy/lazy.nvim/lua/lazy/core/loader.lua:362: in function <...local/share/nvim/lazy/lazy.nvim/lua/lazy/core/loader.lua:360>
	[C]: in function 'xpcall'
	.../.local/share/nvim/lazy/lazy.nvim/lua/lazy/core/util.lua:113: in function 'try'
	...local/share/nvim/lazy/lazy.nvim/lua/lazy/core/loader.lua:377: in function 'config'
	...local/share/nvim/lazy/lazy.nvim/lua/lazy/core/loader.lua:344: in function '_load'
	...local/share/nvim/lazy/lazy.nvim/lua/lazy/core/loader.lua:187: in function 'load'
	...hare/nvim/lazy/lazy.nvim/lua/lazy/core/handler/event.lua:83: in function <...hare/nvim/lazy/lazy.nvim/lua/lazy/core/handler/event.lua:72>
	[C]: in function 'nvim_exec_autocmds'
	.../.local/share/nvim/lazy/lazy.nvim/lua/lazy/core/util.lua:152: in function <.../.local/share/nvim/lazy/lazy.nvim/lua/lazy/core/util.lua:147>

# stacktrace:
  - /plenary.nvim/lua/plenary/job.lua:107 _in_ **new**
  - /laravel.nvim/lua/laravel/runners/async.lua:28
  - /laravel.nvim/lua/laravel/application/init.lua:85 _in_ **run**
  - /laravel.nvim/lua/laravel/commands.lua:48 _in_ **load**
  - /laravel.nvim/lua/laravel/application/init.lua:65 _in_ **warmup**
  - /laravel.nvim/lua/laravel/init.lua:26 _in_ **setup**
  - ~/.config/nvim/lua/plugins/core.lua:162 _in_ **config**`

**Comment by adalessa** [comment]
Hi, from the error is failing to get call the `docker` command.
This could be because it detected that your project should be run with docker, this could be due having a `docker-compose.yaml` file in the directory.
Is that correct for your project ?

**Reply by guiwestrup** [reply]
I don't know the cause, but my mistake was having configured in astronvim without using the user folder.

I've already solved it, thank you very much for your attention.

Thank you very much for the plugin!

**Reply by adalessa** [reply]
Cool that is working. I am not familiar with astrovim, but I am glad that is solved, and you like the plugin.
